### PR TITLE
Added venv at the beggining of the prompt

### LINKED ISSUE #3: Add python venv support
I noticed the other day that the prompt doesn't include the activated python virtual environment. There should be lots of examples on how to do this.

### PR DESCRIPTION
**Pull request by Veinar**
I think it resolves #3.
Please check if solution fit your requirements and merge if everything is all right.

This is my first commitment to other people work, please be kind 🙂 
Thank you,

Regards,
Konrad

### CODE REVIEW

**Comment by miccou** [comment]
Nice work @Veinar , thankyou for your contribution. And congratulations on your first PR here on GitHub!

I made some changes to [miccou/sensa-theme:master](https://github.com/miccou/sensa-theme) to allow easy development and testing of the theme in GitHub codespaces, then I tested your changes and this is what I saw:

image

So it looks good, thanks for adding the functionality. Do you know why the colour isn't showing up correctly? Seems like you picked a blueish colour but it's showing as the default colour for me.

**Comment by Veinar** [comment]
Dear @miccou 

I changed way colors are being picked. Now not longer using `%{$FG[32]%}` and migrated to `%F{32}`.
An issue with colors being not updated in flight is strange. ~For now I cannot tell you why this is happening,~ I assume that checking is being made correctly because `venv-name` displays properly - *see EDIT1 at the bottom*.
```
if [ ! -z "${VIRTUAL_ENV}" ]; then
    prompt_venv="(${VIRTUAL_ENV##*/}) "
else
    prompt_venv=""
fi
```
~but color declared in prompt is not being updated. Which is mutually exclusive case 🤔~ 

~I've got idea to write `precmd()` or `preexec()` in but it didn't worked either way - maybe I didn't this properly~ (no, sourcing `~/.zshrc` such way won't affect currently used shell)
But take look what happens after reexecuting `test.sh` in Workspaces:

obraz

Color of `venv` indicator is displaying properly. And what this script does is: `source ~/.zshrc` magically this updates color and after I recreate this command color is displaying 🤔 

obraz


So to summarize there is a need to somehow update `source ~/.zshrc` at/after command execution.

Regards,
Konrad

P.S: I suppose am not much experienced with `zsh` to solve this issue but I did what I could to help investigate this 😞 I might try to investigate more but maybe you have more knowledge about `zsh` how to move this configuration.

---
**EDIT1:**
By default uncolored `venv name` is being displayed due to built in `venv` mechanisms, we can disable it using
`export VIRTUAL_ENV_DISABLE_PROMPT=1` but after that `venv` name won't be displayed without executing `source ~/.zshrc`

**Comment by Veinar** [comment]
A "non-elegant" way out of this situation is to declare an alias that will cause `source ~/.zshrc` to be executed every time `source` is called.
To the best of my knowledge, you would need to declare something like that:
```
alias source='(){source $@ && source ~/.zshrc}'
```
It will assure that `~/.zshrc` will be always taken as last parameter and coloring should be done correctly 🙂 

Creating such an **alias** would most likely have to be added to the `sensa.zsh-theme` file, which is executed when switching to the `sensa` theme. Or it might me directly appended to `~/.zshrc` in some other way.

**But as I said, this is not an elegant solution, so I just leave it here as a suggestion**

Regards,
Konrad

**Comment by miccou** [comment]
I'll merge this PR - the venv name is showing as expected.

The colour can be fixed in the future.

Thanks for your contribution.

**Comment by Veinar** [comment]
Thank you, sorry I am unable to help further.

Regards,
Konrad
